### DISCUSSION TITLE
Ship heat ratings.

### DISCUSSION THREAD

**Discussion by DioTheCreative** [tsunamayo/Starship-EVO#4194] [Suggestions]
One thing that would be very helpful, is to have a way to see how well your ship dissipates heat. Elite Dangerous has a good method of showing it. The ships ability to dissipate heat depends on how much heat the ship is generating.

The ship has 100% effective heat dissipation when the heat meter reaches max, 0% heat dissipation when the meter is close to zero.

So kinda how Elite works if that explains it better.

**Comment by Briaireous** [comment]
you can check that heat dissipation rate in the ship core and in the entity portion of the tab menu

**Reply by DioTheCreative** [reply]
I know about that, I'm suggesting that it can be shown similar to how elite dangerous does it.
